Automattic/jetpack
# Boost: Fix a regression where speed scores refresh wasn't debouncing

## Proposed changes:

* Some TS fixes
* Fix score refresh debounce

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
1. Open the network panel of your browser.
2. Toggle a module and instantly toggle back to the previous state (within 2 seconds).
3. Make sure there is no call to `speed-score/refresh` in the network panel.
4. Toggle a module without turning it back unlike step 2.
5. There should be a refresh call now.
6. If critical CSS is generating, there will not be any calls to refresh.
